test/integration/reconnect/postgresContainer.js

```diff
@@ -0,0 +1,92 @@
+const Promise = require('bluebird');
+const _        = require('lodash');
+
+function PostgresContainer(docker, options) {
+  var name     = _.get(options, 'name', 'knex-postgres');
+  var image    = _.get(options, 'image', 'postgres:9.6');
+  var username = _.get(options, 'username', 'postgres');
+  var password = _.get(options, 'password', '');
+  var hostPort = _.get(options, 'hostPort', 5432);
```

Comment:
Oh, OK. `5432` was just the default value. The docker container was actually running on `hostPort: 49152`. You can this configuration now on `knexfile.js`.